packages/react-storage/src/components/StorageBrowser/views/LocationsView/LocationsView.tsx

```diff
@@ -103,6 +103,7 @@ export function LocationsView({
 
   // TODO: add hasExhaustedSearch + query param
```

Comment:
Is this todo something we can address here as well?

Reply:
I think that can be a separate PR where we integrate replace the existing view based search with the handler based one